test/cfc/framework/vue3/Vue3FixtureRenderer.tsx

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { defineComponent, h, resolveComponent } = require("vue");
```

Comment:
Is there any reason why only vue doesn't use import?

Reply:
Actually, that was needed at some moment that I can't remember.
But, looks like changing that doesn't trigger any test error. I'll change that to normal esm import.